### DISCUSSION TITLE
Profile private setting

### DISCUSSION THREAD

**Discussion by karol767** [community/community#129556] [Other Feature Feedback, Questions, & Ideas] [answered]
### Profile Customization

Recently, I created new account for freelancing.
But as you can see, I have no work history.
Now I wish to show only my badges and to hide rest actions history.
I know about profile private setting. but in this case, everything is hidden except readme file.
Anyone who can help me?

**Comment by davevad93** [comment]
Hi @karol767 ,

As far as I know it isn't possible to hide profile activity without also hiding achievements, you can hide your private contributions and the activity overview, but the main contributions graph will still be visible.

![0](https://github.com/community/community/assets/94308234/519b227a-70ca-400e-b54c-0c48aa351053)

You can make your profile private, but as you know, that hides everything, therefore it's not possible to accomplish what you're looking for.

**Reply by karol767** [reply]
> Hi @karol767 ,
> 
> As far as I know it isn't possible to hide profile activity without also hiding achievements, you can hide your private contributions and the activity overview, but the main contributions graph will still be visible.
> 
> ![0](https://private-user-images.githubusercontent.com/94308234/342297861-519b227a-70ca-400e-b54c-0c48aa351053.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMzQ4MjgsIm5iZiI6MTcxOTIzNDUyOCwicGF0aCI6Ii85NDMwODIzNC8zNDIyOTc4NjEtNTE5YjIyN2EtNzBjYS00MDBlLWI1NGMtMGM0OGFhMzUxMDUzLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI0VDEzMDg0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlNzM3NTBmMmI2MTBjOGYzMzAxZDA4Yzk1YTNkY2JkNTk3MGVjMmU5ZDBjOTNjYWU5ZDA5NTdiN2E3YTJmZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nHHpiKHkKfO76hN-6ksk_oiAZLhTRG1vwzXvEe585D0)
> 
> You can make your profile private, but as you know, that hides everything, therefore it's not possible to accomplish what you're looking for.

Thanks for your reply.

**Reply by davevad93** [reply]
You're welcome @karol767 ! 👍

**Reply by karol767** [reply]
Let follow each other. I wish to learn from you. 
Your profile is awesomeness.

**Comment by Konoha0Maru** [comment]
Yeah, I see.
but I agree with @davevad93.
In my opinion, To upload your cv or social information and to make a good profile is ideal choice.

**Reply by karol767** [reply]
Thanks. but Is it safe to link my social information?
I will try to upgrade my profile.

**Reply by karol767** [reply, marked as answer]
On my profile page, If I made some public repositories like first merge or first pull request as a private, we can solve this problem mainly.

**Reply by ebndev** [reply]
Hi @karol767, thanks for being a part of the GitHub Community!

I'm glad you were able to find a solution. I took a look at your profile, please feel encouraged to share it and engage with other users in our [Showcase Your GitHub Profile](https://github.com/orgs/community/discussions/121833) thread! 😄

**Reply by karol767** [reply]
Thanks for your advise.
